### DISCUSSION TITLE
Once value selected from autocomplete dropdown not opening on clicking with mouse on it

### DISCUSSION THREAD

**Discussion by rajpootAbubakkar** [trevoreyre/autocomplete#159] [Q&A]
Once i select a value on autocomplete dropdown and my cursor is already on autocomplete input field and i click again on it the dropdown is not opening until i not type anything or i did not click once somewhere else in my app and again click on input field is there any solution for it?

**Comment by andreasgou** [comment]
I have the same problem and it's frustrating for the user not knowing that he has to leave (or type in) the input box for the next click to work.
Strange thing is that when you pick a result by clicking it, the dropdown disappears but if you switch to another tab or application and then you come back, the dropdown appears again.

I actually prefer not to close the dropdown in every result pick, instead to have it visible until the input looses it's focus. It'd be more convenient for the user.
